roles/edpm_pre_adoption_validation/molecule/adoption-negative/molecule.yml

```diff
@@ -0,0 +1,32 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+- command: /sbin/init
+  dockerfile: ../../../../molecule/common/Containerfile.j2
+  image: ${EDPM_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
+  name: edpm-0
+  privileged: true
```

Comment:
nit i also dont think this need to be privileged but i could be wrong

Reply:
removed privilege here and in the other molecule.yml files